python/cudf/cudf/tests/test_orc.py

```diff
@@ -604,13 +604,13 @@ def normalized_equals(value1, value2):
 
 
 @pytest.mark.parametrize("stats_freq", ["STRIPE", "ROWGROUP"])
-@pytest.mark.parametrize("nrows", [1, 100, 6000000])
+@pytest.mark.parametrize("nrows", [1, 100, 100000])
 def test_orc_write_statistics(tmpdir, datadir, nrows, stats_freq):
     from pyarrow import orc
 
     supported_stat_types = supported_numpy_dtypes + ["str"]
     # Can't write random bool columns until issue #6763 is fixed
```

Comment:
@vuule Do you think we should consider putting #6763 back on the queue of things to do? Seems like a bug worth fixing.

Reply:
It's sad that we don't fully support bool columns, but we haven't had any users ask for this (that I know of).
If there's demand, I'll gladly add it to the ~pile~backlog. Not sure if the issue conveys this, but it's not a trivial feature.

Reply:
Can we explicitly disable writing bool columns? This seems like we're writing bad data, and silent corruption isn't something I feel comfortable waiting for users to discover and report.

Reply:
Already done https://github.com/rapidsai/cudf/pull/7261 ;)

Reply:
Ahhhhh. That totally changes my perspective. Can you update the comments to say something like "Writing bool columns exceeding one row group are disabled in libcudf until #6763 is fixed"?

Reply:
Also we should update this test to check that an error is raised in this case, rather than removing the column!

Reply:
This test is for writing statistics, so we really want it to write a table with multiple stripes and verify the written statistics. Letting a test case throw does not contribute to this. 
We do have a separate test for throwing with bool columns (as opposed to silent corruption).

Reply:
Thanks. Apologies, I should have looked first. I have no other concerns.